integration_test/scenarios/chat_detail_scenario.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:linagora_design_flutter/list_item/twake_list_item.dart';
+import '../base/base_scenario.dart';
+import '../robots/chat_group_detail_robot.dart';
+import 'package:flutter/material.dart';
+
+class ChatDetailScenario extends BaseScenario {
+  ChatDetailScenario(super.$);
+
+  Future<void> makeASearch(String searchText) async {
+    await ChatGroupDetailRobot($).getSearchIcon().tap();
+    await $.waitUntilVisible($(AppBar).$(TextField));
+    await $(AppBar).$(TextField).enterText(searchText);
+    // await SearchRobot($).enterSearchText(searchText);
+    await ChatGroupDetailRobot($).waitForEitherVisible($: $, first: $(TwakeListItem), second: $("No Results"), timeout: const Duration(seconds: 10));
+    await Future.delayed(const Duration(seconds: 2));
```

Comment:
You don't want to add the click on a search result and see if it display / scroll to the right message? (We know that today, it doesn't work, maybe that's why you didn't add it here?)

Reply:
please create another test case for searching a message in the past, then open it, then verify the highlight of the message, and the content. please do it in other PR @thuyenh1081